src/activities/CreateAreaMeasurement2D.ts

```diff
@@ -55,7 +52,11 @@ export default class CreateAreaMeasurement2D implements IActivityHandler {
         }
         const mapView = mapProvider.view as MapView;
         let keyDown: ((event: KeyboardEvent) => void) | undefined;
-        let measurement: AreaMeasurement2DResult | undefined;
+        let measurement: {
```

Comment:
You can/should still use `__esri.AreaMeasurement2DViewModelMeasurement | undefined;` here.

Same goes for the other three.

Reply:
Fixed